backend/epub/epub-document.c

```diff
@@ -632,7 +632,7 @@ check_mime_type(const gchar* uri,GError** error)
         const gchar* mimetypes[] = {"application/epub+zip", "application/x-booki+zip", NULL};
         guint i;
 
-        for (i = 0; i < g_strv_length (mimetypes); i++) {
+        for (i = 0; i < g_strv_length ((gchar**) mimetypes); i++) {
```

Comment:
could also be fixed and optimized using
```suggestion
        for (i = 0; mimetypes[i]; i++) {
```

or
```suggestion
        for (i = 0; i < G_N_ELEMENTS (mimetypes); i++) {
```

But that's kind of beyond the scope of this fix.